docs/04-operation-guides/troubleshooting/apix-02-401-unauthorized-403-forbidden.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Cannot connect to a service exposed by an API Rule - 401 Unauthorized or 403 Forbidden
+---
+
+## Symptom
+
+You reach your service and get `401 Unauthorized` or `403 Forbidden` in response.
+
+## Remedy
+
+In such case, make sure that:
+
+- You are using an access token with proper scopes and it is active:
+
+  1. Export the credentials of your OAuth2Client as environment variables:
+
+      ```bash
+      export CLIENT_ID="$(kubectl get secret -n $CLIENT_NAMESPACE $CLIENT_NAME -o jsonpath='{.data.client_id}' | base64 --decode)"
+      export CLIENT_SECRET="$(kubectl get secret -n $CLIENT_NAMESPACE $CLIENT_NAME -o jsonpath='{.data.client_secret}' | base64 --decode)"
+      ```
+
+  2. Encode your client credentials and export them as an environment variable:
+
+      ```bash
+      export ENCODED_CREDENTIALS=$(echo -n "$CLIENT_ID:$CLIENT_SECRET" | base64)
+      ```
+
+  3. Check access token status:
+
+      ```bash
+      curl -X POST "https://oauth2.{DOMAIN}/oauth2/introspect" -H "Authorization: Basic $ENCODED_CREDENTIALS" -F "token={ACCESS_TOKEN}"
```

Comment:
Sorry, should have explained. 
For consistency's sake. `CLUSTER_DOMAIN` is used in Get Started guides and Application Connector documentation, and it's more specific than just `DOMAIN`. I thought it would be good to use the same placeholder throughout the whole documentation.